File: examples/c/constructs/for_loop_continue.c
// Ported from examples/lumen/constructs/for_loop_continue.lm by scripts/port_examples.py; edit the Lumen original, not this file.
int main(void) {
    long i = 0;
    while (i < 11) {
        if (i == 5) {
            i = i + 1;
            continue;
        }
        printf("%ld", i);
        if (i < 10) {
            printf(", ");
        }
        i = i + 1;
    }
    puts("");
    return 0;
}

File: examples/ruby/constructs/for_loop_continue.rb
# Ported from examples/lumen/constructs/for_loop_continue.lm by scripts/port_examples.py; edit the Lumen original, not this file.
for i in 0...11 do
    if i == 5 then
        next
    end
    print(i)
    if i < 10 then
        print(", ")
    end
end
puts("")
